.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -45,6 +45,7 @@ jobs:
       K8S_TEST_HOST_TYPE: "baremetal"
       SNAPSHOTTER: ${{ matrix.snapshotter }}
       PULL_TYPE: ${{ matrix.pull-type }}
+      AUTO_GENERATE_POLICY: "true"
```

Comment:
Changes in this file are for trying to maintain behavior from https://github.com/kata-containers/kata-containers/pull/9385/files#diff-55a1b92d4c9735afcdbe6ef38905599f0f58dbdd7cd5f8db16333529d57a47c3L94-L95